utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -1093,7 +1093,7 @@ int main(int argc, char **argv)
                     2.input bit depth 0 (U8)
                     3.source and destination layout are the same*/
                     if(qaFlag && inputBitDepth == 0 && (srcDescPtr->layout == dstDescPtr->layout) && !(randomOutputCase))
-                        compare_reduction_output(static_cast<Rpp64u *>(reductionFuncResultArr), testCaseName, srcDescPtr, testCase, dst, scriptPath);
+                        compare_reduction_output(static_cast<uint64_t *>(reductionFuncResultArr), testCaseName, srcDescPtr, testCase, dst, scriptPath);
```

Comment:
Why the change from Rpp64u to uint64_t?

Reply:
I had initially missed using abs while comparing present run outputs with reference outputs. 
I got build error

"call to 'abs' is ambiguous"
            int diff = abs(output[i] - refOutput[i]);
Using uint64_t seemed to fix the error